synchronizer/batches.go

```diff
@@ -2,48 +2,47 @@ package synchronizer
 
 import (
 	"context"
-	"encoding/json"
 	"math/rand"
-	"strings"
 	"sync"
 	"time"
 
-	"github.com/0xPolygon/cdk-data-availability/client"
 	"github.com/0xPolygon/cdk-data-availability/config"
 	"github.com/0xPolygon/cdk-data-availability/db"
 	"github.com/0xPolygon/cdk-data-availability/offchaindata"
 	"github.com/0xPolygon/cdk-validium-node/etherman"
 	"github.com/0xPolygon/cdk-validium-node/etherman/smartcontracts/cdkvalidium"
 	"github.com/0xPolygon/cdk-validium-node/jsonrpc/types"
 	"github.com/0xPolygon/cdk-validium-node/log"
-	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/event"
-	"github.com/jackc/pgx/v4"
 )
 
+const blockIterationBatchSize = 128
```

Comment:
I'd recommend making this value a config param. There are RPC providers that limit the size of responses or blocks to be queried